### ISSUE TITLE
Page refreshes when different top topics are selected 

### ISSUE THREAD

**Issue by lerongil** [Qiskit/qiskit-metapackage#125]
Example: Clicking from Overview to Installing Qiskit refreshes the entire page which makes it look likea new page is generated. I believe this is a theme issue.

**Comment by derivation** [comment]
Leron, can you link to an example of another doc site that does not exhibit this behavior? It's not reading to me as a bug.

**Comment by lerongil** [comment]
Here are a few examples of the user experience I would expect: 

https://developer.dji.com/mobile-sdk/documentation/introduction/index.html
https://code.visualstudio.com/docs/editor/integrated-terminal
https://www.tensorflow.org/guide/keras

While some of these do have a slight load time between each page, its negligible in comparison to the qiskit.org/documentation.

**Comment by derivation** [comment]
@diego-plan9 any thoughts on this performance issue?

**Comment by diego-plan9** [comment]
> @diego-plan9 any thoughts on this performance issue?

There is certainly some non-optimal behaviour in the TOC expanding, although rather than performance I think is more a usability issue - will probably be tackled along with #67.

**Comment by carlosazaustre** [comment]
The examples provided are SPAs (Single Page Application). In those kind of websites the page is not "reloading" at route changing because JavaScript (Front-side) is in charge of the routes, and right now we are using a server-side routing.

Sphinx provided a method called `singlehtml´ that @francabrera thinks would be work.

I will work on it.

**Comment by lerongil** [comment]
@carlosazaustre Were you able to get `singlehtml` to work?

**Comment by carlosazaustre** [comment]
I tested the `singleHtml` method and it provides all the content (all the documentation pages) in a single infinite scroll page... So I think is not the best option for now.

**Comment by jaygambetta** [comment]
is this still an issue?

**Comment by francabrera** [comment]
it is still an issue (the pages refreshes when clicking on a menu section), but related to how Sphinx web builder works, so unless someone disagrees I would close it by now since there is not an easy way to fix that other than moving from sphinx and doing something custom

**Comment by lerongil** [comment]
I'm fine with closing but given that it's still an issue maybe just adding the on hold label is more appropriate.

**Comment by SooluThomas** [comment]
I have an ongoing conversation with Salva regarding this issue. I also think on hold is the best option.

**Comment by mtreinish** [comment]
> it is still an issue (the pages refreshes when clicking on a menu section), but related to how Sphinx web builder works, so unless someone disagrees I would close it by now since there is not an easy way to fix that other than moving from sphinx and doing something custom

Has the issue been raised in the sphinx bug tracker? If not you should open an issue about it, in my experience the sphinx community is normally pretty responsive and helpful when someone encounters an issue.

**Comment by SooluThomas** [comment]
I think we can close this issue for now as the issue #467 might be a potential solution for this. Can be reopened if need be.
